src/pytorch_lightning/loggers/neptune.py

```diff
@@ -297,14 +254,10 @@ def __init__(
             self._run_instance[_INTEGRATION_VERSION_KEY] = pl.__version__
 
     def _retrieve_run_data(self) -> None:
-        try:
-            assert self._run_instance is not None
-            self._run_instance.wait()
-            self._run_short_id = self._run_instance["sys/id"].fetch()
-            self._run_name = self._run_instance["sys/name"].fetch()
-        except NeptuneOfflineModeFetchException:
-            self._run_short_id = "OFFLINE"
-            self._run_name = "offline-name"
+        assert self._run_instance is not None
+        self._run_instance.wait()
+        self._run_short_id = self._run_instance["sys/id"].fetch()
+        self._run_name = self._run_instance["sys/name"].fetch()
```

Comment:
Why remove the try and catch here?

Reply:
It's used to set up some private variables in case the user is in connects to Neptune in offline mode. This is a part of the latest neptune-client.

Docs: 
https://docs.neptune.ai/you-should-know/connection-modes#offline

Exception source code:
https://github.com/neptune-ai/neptune-client/blob/master/neptune/new/exceptions.py#L852

Reply:
This code should not be removed since it supports for the actively maintained offline run feature.

Reply:
@shnela please check here.

Reply:
Added back